ReSharper.FSharp/test/data/features/generate/overrides/Anchor - Repr 23.fs

```diff
@@ -0,0 +1,16 @@
+// ${KIND:Overrides}
+// ${SELECT0:ToString():System.String}
+
+// ${KIND:Overrides}
+
+module M
+
+type C(y: int) = class
+
+        do
+            let f{caret} x y = x + y
+            ()
+
+        let x = y
+
+    end
```

Comment:
@dawedawe Could you try to simplify this test a bit, so it shows what exactly is being tested here, please?

It mixes some unrelated things: normalizing the class representation, moving members, the constructor has parameters, the `do` statement is rather complicated and contains inner let binding. It may be a good integration test, but when having many unit tests, like we do here, it would be much easier to reason about each case if it contained the required minimum only, e.g. for checking `do` statements in explicit reprentations:

```fsharp
type T() =
    class
        do (){caret}
    end
```

```fsharp
type T() =
    class
        do (){caret}
    end
```

Or, if checking it next to another member:
```fsharp
type T() =
    class{caret}
        do ()
        member this.P = 1
    end
```

Reply:
Sure, makes sense.